content/en/guides/outbound-oss.md

```diff
@@ -0,0 +1,724 @@
+
+* increase developer satisfaction
+* improve quality and boosts developer skills by peer review of each contribution by core experts
+* make company visible as an attractive employer
+* improve company's reputation, and with it the standing of developers in their communities
+
+### Give back and keep open source sustainable
+
+Open source software development is living from its communities. As mentioned above, the consumption of open source software helps to decrease costs and speed up development, but that's only possible because there is the community behind these projects maintaining the software. To keep the open source development model sustainable, each consumer of open source software has therefore the responsibility to think about ways how to support these projects. These are some ways of engagement and support:
+
+* contributions in terms of code, documentation, time (by testing software, for example)
+* monetary support (some important projects are maintained by developers who do this in their spare time and thus can only invest limited time in the project)
+* executing and supporting hackathons
+
+It is important to understand that though open source software has no license costs when consuming it, it is not available for free. To keep these projects attractive for its consumers, steady engagement and support is required. That's why it is important to have a strategy for open source contributions in place.
+
+# How to contribute to OSS projects
+
+Building better relationships with the open source ecosystem has its own set of challenges, but it becomes easier if you have a clear plan to follow. Here are some guidelines to a number of practices that organizations can adopt.
+
+## Define your open source goal and strategy
+
+Your open source strategy connects the plans for managing, participating in, and creating open source software with the business objectives that the plans serve. This can open up many opportunities and catalyze innovation. The TODO Group offers a dedicated guide to [Setting an Open Source Strategy](https://todogroup.org/guides/strategy/)
+
+## Establish open source guiding principles and processes
+
+### Guiding principles
+
+The procedure described in the following is designed to ensure that the company interests and its employees are protected. We also need to make sure that contributions are in line with copyright law, export regulations, data protection regulations and open source development best practices. On the other hand, the procedural burden for all to be involved stakeholders shall be low and the approval procedure should not take too much time.
```

Comment:
I would think that patent and IP policies are part of the "company interests" which are mentioned in the first sentence. We could spell these out a bit more. I would try to avoid the general term "intellectual property" and be more specific about what kind of rights we are talking here.

Reply:
yes, ok, I agree. Reading through the text again, the items listed in the sentence are actually "external" factors, whereas "IP policies" are "internal" ones to a company - as you mentioned. So, yeah, we can discard this comment.